R/su_chart_theme.R

```diff
@@ -1,71 +1,45 @@
 su_chart_theme <- function(font_family = "Segoe UI") {
-  text_grey <- "#3e3f3a"
-  light_bkg <- "grey95"
-  dark_slate <- StrategyUnitTheme::su_theme_cols("dark_slate")
-  loran_tr <- "#fcdf8377" # su_theme_cols("light_orange") with transparency
-  slate_tr <- "#b2b7b977" # su_theme_cols("light_slate") with transparency
+  slate <- "#686f73"
+  dark_slate <- "#343739"
+  light_slate <- "#b2b7b9"
+  light_red <- "#f5b2aa"
+  light_bg <- "#f5f4f3" # hardcoded, colour comes from  Shiny theme?
   ggplot2::theme(
     text = ggplot2::element_text(
       family = font_family,
       size = 20,
-      colour = text_grey
+      colour = dark_slate
     ),
     title = ggplot2::element_text(hjust = 0.05),
-    plot.title = ggplot2::element_text(
-      size = 28,
-      margin = ggplot2::margin(20, 8, 12, 18)
-    ),
-    plot.subtitle = ggplot2::element_text(
-      margin = ggplot2::margin(2, 0, 12, 20)
-    ),
-    plot.caption = ggplot2::element_text(
-      hjust = 0.96,
-      size = 16,
-      margin = ggplot2::margin(12, 6, 6, 0)
-    ),
-    plot.title.position = "plot",
-    plot.caption.position = "plot",
-    plot.background = ggplot2::element_rect(fill = light_bkg, colour = NA),
-    plot.margin = ggplot2::margin(6, 24, 6, 6),
-    panel.background = ggplot2::element_rect(fill = light_bkg),
+    plot.title = ggplot2::element_text(margin = ggplot2::margin(12)),
```

Comment:
```suggestion
```
If not used - let's delete

Reply:
I *think* this actually gives a margin around axis titles as well, which is good, stops them being squashed against axis labels. Can we leave in for now? I may experiment.